site/docs/connectors/index.md

```diff
@@ -173,6 +174,8 @@ There are currently no archive services supplied by Egeria.
 | [REST Client Connectors](#rest-client-connectors) | issue REST API calls to Egeria's deployed platforms and third party technologies. |
 | [Cohort Member Client Connector](#cohort-member-client-connectors) | supports repository service called to remote cohort members. |
 | [Open Metadata Topic Connectors](#open-metadata-topic-connectors) | send and receive events. |
+| [Open Lineage Janus Connector](/connectors/runtime/open-lineage-janus-connector) | The Open Lineage connectors provide plugins to the [Open Lineage Server](/concepts/open-lineage-server) that allow the Open Lineage Services to connect with databases.
```

Comment:
After reviving the full page content I came across conclusion that it might be better to add new group"Governance Daemon Connectors"  under  "Metadata exchange and maintenance connectors".  (See https://github.com/odpi/egeria/tree/master/open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors for description)

My reasoning: This connector always runs under Open Lineage Server that is governance server with specialized purpose for lineage warehousing that makes it more part of the "additional" metadata maintenance connectors then core runtime connectors.

I suggest following: add 1) new entry under the table "Metadata exchange and maintenance connectors"
and 2) new section "Governance Daemon Connectors" (right after under Governance Action Services). and link the connector page from 3) new folder connectors/governance-daemon and move the /open-lineage-janus-connector there. 

What do you think?

Reply:
Moved them to a new folder, as you suggested